src/gaen/nativeModule.ts

```diff
@@ -140,8 +140,9 @@ export const getVersion = async (): Promise<string> => {
   return deviceInfoModule.getVersion()
 }
 
-export const isBluetoothEnabled = async (): Promise<string> => {
-  return deviceInfoModule.isBluetoothEnabled()
+export const isBluetoothEnabled = async (): Promise<boolean> => {
+  const bluetoothStatus = deviceInfoModule.isBluetoothEnabled()
+  return bluetoothStatus === true || bluetoothStatus === "true"
```

Comment:
I was running into an issue (on Android, idk if this is OS specific), where this function was returning the boolean value `true`, then failing to update the status because `true === "true"` is false.

Reply:
I can change the return type from `Boolean` to `String` in the native side if it's better for you.
I thought iOS was returning a boolean too

Reply:
I personally would prefer to return a `Boolean` rather than a `String`, so I think you made the right call! I'm fine with either keeping this as it is (`status === true || status === "true"`) or updating the iOS to return a boolean rather than a string. What do you think?

Reply:
@amandabeiner I also prefer to use a Boolean instead of an String here.
@mattThousand Is iOS returning a `String` when `isBluetoothEnabled` is called? If so, is it possible to change it to a `Boolean`?